staging/src/k8s.io/apiserver/pkg/apis/audit/types.go

```diff
@@ -207,6 +207,10 @@ type PolicyRule struct {
 	//  "/healthz*" - Log all health checks
 	// +optional
 	NonResourceURLs []string
+
+	// OmitStages specify events generated in which stages will not be emitted to backend.
+	// An empty list means no restrictions will apply.
```

Comment:
should we mark this as optional?

Reply:
Yes, we should.

Reply:
Added here together:
https://github.com/kubernetes/kubernetes/pull/54634